.github/workflows/release.yaml

```diff
@@ -0,0 +1,249 @@
+# This workflow is used to publish SV2 crates to cratesio as well as releasing binaries of the roles
+# (translator, pool, mining_proxy) to github. The github releases will fail if there are no tags available
+# so the `autoversion` workflow must be ran and the resulting PR must be merged at least once before this will 
+# release the binaries to github. 
+
+# If the binary releases do fail to due to not having tags, force run the `autoversion` workflow
+# on the main branch and merge the resulting PR to create the tags and move them to the main branch.
+
+# This should be disabled for forks
+
+name: Release
+
+on: 
+  # Manually run by going to "Actions/Release" in Github and running the workflow
+  workflow_dispatch:
+
+jobs:
+  # Cannot run until we have necessary keys to publish crates to crates.io.
+  # This should not be run on forks since removing the `--no-changelog` flag in the "Commit version updates"
+  # step will create all changelog files with respect to the fork this was executed in, so each changlog file
+  # will log the forks history of the crate instead of the main repos history
+  crates_publish:
+    name: Release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: 
+          - ubuntu-latest
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-musl
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: 
+        run: echo TEST
+      
+      # - name: Install cargo-smart-release
```

Comment:
Can you remove this commented out block or add another comment as to why we want to keep it here?